web/modules/custom/collection/src/Form/JoinCollectionForm.php

```diff
@@ -50,26 +50,48 @@ public function buildForm(array $form, FormStateInterface $form_state, AccountPr
     // confirmation form, disguised as a form submit button. The confirmation
     // form should open in a modal dialog for JavaScript-enabled browsers.
     if (Og::isMember($collection, $user)) {
-      $form['leave'] = [
-        '#type' => 'link',
-        '#title' => $this->t('Leave this collection'),
-        '#url' => Url::fromRoute('collection.leave_confirm_form', [
-          'rdf_entity' => $collection->id(),
-        ]),
-        '#attributes' => [
-          'class' => [
-            'use-ajax',
-            'button',
-            'button--small',
-            'button--blue-light',
-            'mdl-button',
-            'mdl-js-button',
-            'mdl-button--raised',
-            'mdl-js-ripple-effect',
-            'mdl-button--accent',
+      $form['leave_wrapper'] = [
+        '#type' => 'dropbutton',
```

Comment:
Hi @tom-waat 
I synced with Wesley for this:
- the whole button should be clickable
- the MDL dropdown should be used:

![correct_animation](https://user-images.githubusercontent.com/1923476/28625345-36ae14bc-721c-11e7-91e8-9917c4c04e30.gif)